.env.example

```diff
@@ -10,10 +10,10 @@
 
 WEBAPP_URL=http://localhost:3000
 
-SURVEY_BASE_URL=http://localhost:3000/i
+SURVEY_BASE_URL=http://localhost:3000/s
 
 # Set this if you want to have a shorter link for surveys
-SHORT_SURVEY_BASE_URL=
+SHORT_SURVEY_BASE_URL=http://localhost:3000/i
```

Comment:
Please keep this empty be default as if it's empty, it wont be considered and the SURVEY_BASE_URL will work